### ISSUE TITLE
Proxy Configuration

### ISSUE THREAD

**Issue by Serhatcck** [ticarpi/jwt_tool#104]
Hi there! Firstly, thank you for your project. It's really useful and great! In my opinion, if you could incorporate proxy configuration in cmd, it would be very helpful for WSL users. This is because the proxy might be running on a different host from my WSL/Linux distribution, which has a different IP address. This could lead to issues when using the loopback interface.

**Comment by Elitesid** [comment]
yes I'm facing the same issue as @Serhatcck. It would be great if this issue is addressed by dev

**Comment by huornlmj** [comment]
+1 on proxy support.
